authorization/config.yaml

```diff
@@ -32,6 +31,7 @@ apis:
       clientCertificateRequired: true
 oauthClients:
   device:
+    ownerClaim: sub
```

Comment:
It doesn't belong to oauthClients.device. When we move this configuration to coapgateway, you will have to add it to other place in this authorization-server config - again breaking change. And, coap-gateway doesn't need it anyway. It's the general configuration of the authorization server.

CoAP gateway just sends "register device" request to authorization server with token, it won't care about jwt claims.

Reply:
As discussed, will be moved to storage group.